src/vehicle.cpp

```diff
@@ -3242,15 +3244,18 @@ point vehicle::pivot_displacement() const
 
 int vehicle::fuel_left( const itype_id &ftype, bool recurse ) const
 {
-    int fl = std::accumulate( parts.begin(), parts.end(), 0, [&ftype]( const int &lhs,
-    const vehicle_part & rhs ) {
-        // don't count frozen liquid
-        if( rhs.is_tank() && !rhs.base.contents.empty() &&
-            rhs.base.contents.legacy_front().made_of( phase_id::SOLID ) ) {
-            return lhs;
+    int fl = 0;
+
+    for( size_t i = 0; i < fuel_containers.size(); ++i ) {
```

Comment:
```suggestion
    for( const size_t i : fuel_containers ) {
        const vehicle_part &part = parts[i];
```
right?  You're just looping through fuel_containers to get the index value into parts.

Reply:
Yes, there is this one and bunch of other issues that are failing clang-tidy build. I'll try to fix them all today.